.github/guides/STYLE.md

```diff
@@ -211,6 +211,9 @@ While DM allows other ways of declaring variables, this one should be used for c
 ### Use descriptive and obvious names
 Optimize for readability, not writability. While it is certainly easier to write `M` than `victim`, it will cause issues down the line for other developers to figure out what exactly your code is doing, even if you think the variable's purpose is obvious.
 
+#### Any time variable or argument that uses a unit of time other than decisecond must include the unit of time in the name.
```

Comment:
```suggestion
#### Any variable or argument that uses a unit of time other than decisecond must include the unit of time in the name.
```

i feel like that first time is a bit redundant given how many other times (haha) we mention time in this title, kinda takes me out of it. were you meaning something like:

```suggestion
#### Any time a variable or argument that uses a unit of time other than decisecond must include the unit of time in the name.
```

Hard to tell

Reply:
it means any variable that holds time, time variable, so its still needed.

Reply:
`All time variables or arguments that use a unit of time...` may be more clear.